types/ws/index.d.ts

```diff
@@ -82,7 +82,7 @@ declare namespace WebSocket {
 
     type VerifyClientCallbackSync = (info: { origin: string; secure: boolean; req: http.IncomingMessage }) => boolean;
     type VerifyClientCallbackAsync = (info: { origin: string; secure: boolean; req: http.IncomingMessage }
-        , callback: (res: boolean) => void) => void;
+        , callback: (res: boolean, code?: number, message?: string) => void) => void;
```

Comment:
```typescript
const wss = new WebSocket.Server({
    host: address,
    port: port,
    verifyClient: (info, verify) => {
        const id = getIdFromRequest(info.req);  // Get "id" param from URL parameters.
        if (id) {
            verify(true);
        } else {
            verify(false);
        }
    }
});
```

Suppose we have existing code like above, it works with current declaration.

But if we extend the declaration without optional, just like:

```typescript
callback: (res: boolean, code: number, message: string) => void) => void;
```

The tsc compiler will reject the code with error `[ts] Supplied parameters do not match any signature of call target.`